certification/internal/shell/scorecard_basic-check-spec.go

```diff
@@ -0,0 +1,123 @@
+package shell
+
+import (
+    "os/exec"
+    "os"
+    "io/ioutil"
+    "path/filepath"
+
+    "github.com/komish/preflight/certification"
+    "github.com/sirupsen/logrus"
+)
+
+type ScorecardBasicSpecCheck struct{}
+
+func (p *ScorecardBasicSpecCheck) Validate(bundle string, logger *logrus.Logger) (bool, error) {
```

Comment:
Does `bundle` refer to `bundleImage`? It would be great to have a more descriptive name :)

Reply:
I corrected.